src/plugin.c

```diff
@@ -281,43 +288,20 @@ int plugin_score(int band, const char *call, int mode,
     return result;
 }
 
-void plugin_add_qso(const char *logline) {
-#ifdef HAVE_PYTHON
-//    if (pf_add_qso == NULL) {
-//        return;
-//    }
-//    printf("--- in add_qso |%s|\n", logline);
-// parse: call, ...
-    char call[14];
-    g_strlcpy(call, logline + 29, sizeof(call));
-    g_strchomp(call);
-    int mode = log_get_mode(logline);
-    int band = log_get_band(logline); //atoi(logline);
-
-    PyObject *qso = create_py_qso(band, call, mode, "");
-
-    // call add_qso
-    PyObject *arg = Py_BuildValue("(O)", qso);
-    PyObject *pValue = PyObject_CallObject(pf_add_qso, arg);
-    Py_DECREF(arg);
-    Py_DECREF(qso);
-
-    if (NULL != PyErr_Occurred()) {
-	PyErr_Print();
-	//FIXME: action?
-    }
-    Py_XDECREF(pValue);
-#endif
-}
-
 bool plugin_is_multi(int band, const char *call, int mode) {
```

Comment:
Is it necessary the `const` modifier here? The `call` member of `qso_t` is [seems](https://github.com/Tlf/tlf/blob/master/src/tlf.h#L204) like just a single `char *`. Both `gcc` and `clang` generated a warning. I think we should leave it.